### PR TITLE
Docs/ee get started

### PR DESCRIPTION
**Pull request by coopr**
### Summary

Add Getting Started docs to EE by copying from CE and editing

### Full changelog

* Copied "Getting Started" docs from CE to EE
* Edited docs to reflect EE

### Checklist:

- [x] Spellchecked my documentation
- [x] Documentation 
- [x] Ready to be merged 
Not ready to merge because I was unable to run locally, thus I didn't test these changes.

### CODE REVIEW

File: app/docs/enterprise/0.31-x/getting-started/adding-consumers.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+title: Adding Consumers
+---
+
+# Adding Consumers
+
+<!-- <div class="alert alert-warning">
```
Comment (travispamaral) [inline app/docs/enterprise/0.31-x/getting-started/adding-consumers.md]: we should remove this if we don't plan on showing it

File: app/docs/enterprise/0.31-x/getting-started/adding-consumers.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  <strong>Before you start:</strong>
```
Comment (travispamaral) [inline app/docs/enterprise/0.31-x/getting-started/adding-consumers.md]: same here

File: app/docs/enterprise/0.31-x/getting-started/adding-your-api.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
+---
+title: Adding your API
+---
+
+# Adding your API
+
+<!-- <div class="alert alert-warning">
```
Comment (travispamaral) [inline app/docs/enterprise/0.31-x/getting-started/adding-your-api.md]: same here

File: app/docs/enterprise/0.31-x/getting-started/enabling-plugins.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
+---
+title: Enabling Plugins
+---
+
+# Enabling Plugins
+
+<!-- <div class="alert alert-warning">
```
Comment (travispamaral) [inline app/docs/enterprise/0.31-x/getting-started/enabling-plugins.md]: ""

File: app/docs/enterprise/0.31-x/getting-started/introduction.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+---
+title: Welcome to Kong Enterprise Edition
+---
+
+# Welcome to Kong Enterprise Edition
+
+<!-- <div class="alert alert-warning">
```
Comment (travispamaral) [inline app/docs/enterprise/0.31-x/getting-started/introduction.md]: ""

File: app/docs/enterprise/0.31-x/getting-started/quickstart.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
+---
+title: 5-minute Quickstart
+---
+
+# 5-minute Quickstart
+
+<!-- <div class="alert alert-warning">
```
Comment (travispamaral) [inline app/docs/enterprise/0.31-x/getting-started/quickstart.md]: ""

File: app/docs/enterprise/0.31-x/getting-started/adding-consumers.md (hunk 1 above)
Comment (coopr) [reply app/docs/enterprise/0.31-x/getting-started/adding-consumers.md]: Well, we want to show it, but it need to link to a page that doesn't exist yet... shall I remove it until that other page exists?

File: app/_data/docs_nav_ee_0.31-x.yml [hunk 7]
```diff
@@ -1,5 +1,15 @@
 - title: Getting Started
   items:
+     - text: Introduction
```
Comment (travispamaral) [inline app/_data/docs_nav_ee_0.31-x.yml]: This has an extra space before the `-` causing it to error and not render

File: app/_data/docs_nav_ee_0.31-x.yml [hunk 8]
```diff
@@ -1,5 +1,15 @@
...
+      url: /getting-started/introduction
+     - text: Five-minute quickstart
```
Comment (travispamaral) [inline app/_data/docs_nav_ee_0.31-x.yml]: same as above

**Pull request merged**